x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -270,6 +280,10 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
           timestamp: indexedStartedDate,
           spaceId,
         }),
+        originalAlertState,
+        originalAlertStateWasALERT: originalAlertState === stateToAlertMessage[AlertStates.ALERT],
```

Comment:
Why do we compare the translated state values instead of the original ones. Wouldn't that be more robust in case the Kibana language is changed?